parlai/core/torch_ranker_agent.py

```diff
@@ -72,15 +72,14 @@ def __init__(self, opt, shared=None):
                 states = {}
 
         self.rank_loss = nn.CrossEntropyLoss(reduce=True, size_average=False)
+        if self.use_cuda:
```

Comment:
I might want to put that in another PR. I think model.cuda() should be called before  self.set_fixed_candidates(shared) since this function is using self.model

Reply:
I was gonna ask about this... seems okay.